cromulent/multiple_instantiation.py

```diff
@@ -1,30 +1,43 @@
 
-# This assumes the default CIDOC-CRM, even though the model code 
+# This assumes the default CIDOC-CRM, even though the model code
 # can generate classes for any ontology
 
 import inspect
-from .model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject	
+
+# gvp 'from model' instead 'from .model'
+from model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject
+#from .model import Destruction, EndOfExistence, Activity, Appellation, LinguisticObject
+##
 
 # DestuctionActivity class as CRM has a Destruction Event and recommends multi-classing
 # WARNING:  instantiating this class in the default profile will raise an error
 
+
 class DestructionActivity(Destruction, Activity):
-	_uri_segment = "Activity"
-	_type = ["crm:E6_Destruction", "crm:E7_Activity"]
-	_niceType = ["Destruction", "Activity"]
+    _uri_segment = "Activity"
```

Comment:
Is this change to replace a tab with four spaces? (Hard to see in the github UI)